upgrade-guide.md

```diff
@@ -9,6 +9,26 @@ TODO
 
 ### Breaking Changes
 
+- **Solid Facets & Surface Edges**: Edge/Facet indices are used as parameters of methods like `PolygonalSurface< dimension >::edge_length`, `PolyhedralSolid< dimension >::facet_barycenter`.
+
+**How to upgrade**
+
+Example
+
+from
+
+```
+PolygonEdge polygon_edge{ 0, 0 };
+auto edge_length = surface.polygon_edge_length( polygon_edge );
+```
+
+to 
+
+```
+index_t edge_id = polygon_edge( { 0, 0 } );
```

Comment:
it could be auto

Reply:
I agree with you but I explicitly write index_t to show that edge_length() takes an index_t. What is your opinion?

Reply:
ok